website/source/layouts/aws.erb

```diff
@@ -10,6 +10,39 @@
                     <a href="/docs/providers/aws/index.html">AWS Provider</a>
                 </li>
 
+                <li<%= sidebar_current(/^docs-aws-resource-apigateway/) %>>
+                    <a href="#">APIGateway Resources</a>
+                    <ul class="nav nav-visible">
+                        <li<%= sidebar_current("docs-aws-resource-api-gateway-rest-api") %>>
+                            <a href="/docs/providers/aws/r/api_gateway_rest_api.html">aws_api_gateway_rest_api</a>
+                        </li>
+                        <li<%= sidebar_current("docs-aws-resource-api-gateway-deployment") %>>
+                            <a href="/docs/providers/aws/r/api_gateway_deployment.html">api_gateway_deployment</a>
```

Comment:
The link title here should be `aws_api_gateway_deployment ` (including the `aws_`).